src/azure-cli-core/azure/cli/core/style.py

```diff
@@ -72,3 +72,22 @@ def format_styled_text(styled_text):
     # Reset control sequence
     formatted_parts.append(Fore.RESET)
     return ''.join(formatted_parts)
+
+
+def get_styled_command(raw_command):
```

Comment:
How about we add some function doc and give some examples about the behavior?

Reply:
Sure, moved to #16349